tests/fault_injector/test_fault_configs.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+
+class TestPlatformBlockedFaultConfiguration:
```

Comment:
Where do you use the fixtures? 🤔

Reply:
Ah right, these are obsolete now. Thank you for mentioning it!